col/src/main/java/vct/col/ast/generic/ASTNode.java

```diff
@@ -418,7 +418,7 @@ public ASTList annotations(){
   }
   
   public boolean annotated(){
-    return annotations!=null && annotations.size()>0; 
+    return annotations!=null;
```

Comment:
I'll give it a shot! I don't think we should distinguish "absence of annotations" and "zero annotations" anyway.

Reply:
Ah now I remember, whether you've "touched" the annotations matters in AbstractRewriter, which only copies the annotations automatically if the annotations are `null` (not empty). Otherwise we would re-add the annotations after `AnnotationInterpreter`. I don't think this is a great design, but I don't want to refactor it right now.